evalml/tests/automl_tests/test_default_algorithm.py

```diff
@@ -727,3 +728,69 @@ def test_default_algorithm_time_series_known_in_advance(
     long_estimators = set([pipeline.estimator.name for pipeline in long_2])
     assert len(long_2) == 30
     assert len(long_estimators) == 3
+
+
+@pytest.mark.parametrize("split", ["numeric-only", "categorical-only"])
```

Comment:
Still need to figure out implementation and test for the "split" pipeline case. It'll be a little more confusing as I would need to figure out how to do the follow pipeline structure:

                               /  ---------------[non-categorical pipeline]----------------------\
[DFSTransformer]                                                                                                               [estimator]
                               \ ---------------[categorical pipeline]---------------------------/